ptmdmusique/cs464-564-final-project
# feat: add randomly generated pokemon id list

## Describe your changes

- So previously the toggle didn't work with a small number of Pokemon because we fetch the Pokemons based on continuous IDs
- With a randomly generated ID list, we can make the toggle work for the demo while not getting IP blocked ourselves...
- I tried to copy all 1010 data into a TS file but it turns out to be soooooooooooooooooooooo big and crashed my IDE... so that was not a good idea haha

## Step to test



- Make sure the `/statistics` page load properly 
- Make sure the Regions toggle works

## Issue ticket number and link



-

## Checklist before requesting a review
